Makefile

```diff
@@ -71,15 +78,20 @@ check: python-env
 # Corrects spelling errors
 .PHONY: misspell
 misspell:
-	go get github.com/client9/misspell
-	${FIND} -name '*'  -exec misspell -w {} \;
+	go get $(MISSPELL_REPO)
```

Comment:
I tend to prefer doing this so the actual logic is more readable and easier to maintain (ideally also have `$(GO)`). If you don't like it I can remove it though.